src/components/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  newTodoField: React.RefObject<HTMLInputElement>;
+  addNewTodo: (title: string) => void;
+  toggleAllTodosStatus: () => void;
+  isAllTodosCompleted: boolean;
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  newTodoField,
+  addNewTodo,
+  toggleAllTodosStatus,
+  isAllTodosCompleted,
+}) => {
+  const [title, setTitle] = useState('');
+
+  const onSumbit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    addNewTodo(title);
+    setTitle('');
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  return (
+    <header className="todoapp__header">
+      {(!!todos.length) && (
```

Comment:
No need `()`, as for me: 
```suggestion
      {!!todos.length && (
```